docs/openapi/saved_objects/saved_objects.yml

```diff
@@ -399,6 +399,38 @@ paths:
             application/json:
               schema:
                 type: object
+  /api/saved_objects/_bulk_get:
+    get:
+      tags:
+        - saved objects
+      summary: Bulk get saved objects
+      requestBody:
+        required: true
+        description: Array of saved objects to update
```

Comment:
To update or to retrieve? The array is search criteria, right?

Reply:
THanks for pointing this out. Will make the change